### DISCUSSION TITLE
Pasting rich-text-format content into Athens results in PNG image instead of text

### DISCUSSION THREAD

**Discussion by yifanyin** [athensresearch/athens#836] [Archive]
I am guessing it's the problem with RTF? This happens when I copy-paste from Microsoft Word documents. This also happens to users copy-pasting from OneNote.

**Comment by jsmorabito** [comment]
a temporary workaround for this, provided by danield, is the following:

Paste your content as plain text:

on Mac:
CMD+OPTION+SHIFT+v 

on Windows:
CTRL+SHIFT+v
